hearpreprocess/nsynth_pitch.py

```diff
@@ -48,16 +54,16 @@
     "modes": {
         "5h": {
             "max_task_duration_by_split": {
-                "train": 3600 * 5 * 3 / 4,
-                "valid": 3600 * 5 * 1 / 4,
-                "test": 3600 * 5 * 1 / 4,
+                "train": 3600 * 5 * TRAIN_PERCENTAGE / TRAINVAL_PERCENTAGE,
+                "valid": 3600 * 5 * VALIDATION_PERCENTAGE / TRAINVAL_PERCENTAGE,
+                "test": 3600 * 5 * TEST_PERCENTAGE / TRAINVAL_PERCENTAGE,
```

Comment:
I'm confused. These percentages add up to over 100%. If I understand, we are saying the train and valid should add up to 5hrs? Maybe this should be documented.